src/server/svr_movejob.c

```diff
@@ -702,200 +672,6 @@ int
-	}
-
-	/* in windows code, a child process "w32_send_job" handles the send
-	 * This needs the job information, so we save using the filesystem
-	 * This avoids the child process from having to "connect" to the database again
-	 * The file is deleted by the send_job child process when it has done recovering the job
-	 */
-	job_save_fs(jobp, SAVEJOB_FULLFORCE);	/* so the spawned process can get a fresh copy of job */
-
-	if (*jobp->ji_qs.ji_fileprefix != '\0')
-		sprintf(buf, "jobfile=%s%s\n", jobp->ji_qs.ji_fileprefix, JOB_FILE_SUFFIX);
-	else
-		sprintf(buf, "jobfile=%s%s\n", jobp->ji_qs.ji_jobid, JOB_FILE_SUFFIX);
-
-	win_pwrite(&pio, buf, strlen(buf));
-
-	sprintf(buf, "destaddr=%ld\n", hostaddr);
-	win_pwrite(&pio, buf, strlen(buf));
-
-	sprintf(buf, "destport=%d\n", port);
-	win_pwrite(&pio, buf, strlen(buf));
-
-	sprintf(buf, "move_type=%d\n", move_type);
-	win_pwrite(&pio, buf, strlen(buf));
-
-	sprintf(buf, "in_server=%d\n", is_linked(&svr_alljobs, &jobp->ji_alljobs));
-	win_pwrite(&pio, buf, strlen(buf));
-
-	sprintf(buf, "server_name=%s\n", (server_name?server_name:""));
-	win_pwrite(&pio, buf, strlen(buf));
-
-	sprintf(buf, "server_host=%s\n", (server_host?server_host:""));
-	win_pwrite(&pio, buf, strlen(buf));
-
-	sprintf(buf, "server_addr=%ld\n", pbs_server_addr);
-	win_pwrite(&pio, buf, strlen(buf));
-
-	sprintf(buf, "server_port=%d\n", pbs_server_port_dis);
-	win_pwrite(&pio, buf, strlen(buf));
-
-	sprintf(buf, "log_file=%s\n", (log_file?log_file:""));
-	win_pwrite(&pio, buf, strlen(buf));
-
-	sprintf(buf, "path_log=%s\n", (path_log?path_log:""));
-	win_pwrite(&pio, buf, strlen(buf));
-
-	sprintf(buf, "path_jobs=%s\n", (path_jobs?path_jobs:""));
-	win_pwrite(&pio, buf, strlen(buf));
-
-	sprintf(buf, "path_spool=%s\n", (path_spool?path_spool:""));
-	win_pwrite(&pio, buf, strlen(buf));
-
-	sprintf(buf, "path_rescdef=%s\n", (path_rescdef?path_rescdef:""));
-	win_pwrite(&pio, buf, strlen(buf));
-
-	sprintf(buf, "path_users=%s\n", (path_users?path_users:""));
-	win_pwrite(&pio, buf, strlen(buf));
-
-	sprintf(buf, "path_hooks_workdir=%s\n",
-		(path_hooks_workdir?path_hooks_workdir:""));
-	win_pwrite(&pio, buf, strlen(buf));
-
-	sprintf(buf, "svr_history_enable=%ld\n", svr_history_enable);
-	win_pwrite(&pio, buf, strlen(buf));
-
-	sprintf(buf, "svr_history_duration=%ld\n", svr_history_duration);
-	win_pwrite(&pio, buf, strlen(buf));
-
-	if ( (server.sv_attr[SRV_ATR_ssignon_enable].at_flags & \
-                                                ATR_VFLAG_SET) && \
-             (server.sv_attr[SRV_ATR_ssignon_enable].at_val.at_long == 1) )
-		strcpy(buf, "single_signon_password_enable=1\n");
-	else
-		strcpy(buf, "single_signon_password_enable=0\n");
-
-	win_pwrite(&pio, buf, strlen(buf));
-
-	sprintf(buf, "script_name=%s\n", script_name);
-	win_pwrite(&pio, buf, strlen(buf));
-
-	strcpy(buf, "quit\n");
-	win_pwrite(&pio, buf, strlen(buf));
-	win_pclose2(&pio);	/* closes all handles except the process handle */
-	return (2);
-#else
```

Comment:
Shouldn't there be a removal of an endif that goes with the removal of this else?

Reply:
Yes @bhroam, that is removed at the end of send_job function:
https://github.com/PBSPro/pbspro/pull/1414/files/efa51fb0c8837da4d6c3953a7895138f83ee28d3#diff-a51b4b3581fd3b2b0856a1ae6763b1f4L1269